templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
```

Comment:
I think we should wait until we get a proper name for this component as we've already changed names 4 times and it'd be good to get a finalised name before updating it across the templates.